autoload/twiggy.vim

```diff
@@ -84,9 +84,10 @@ let s:icons.unmerged = s:icon_set[6]
 " {{{1 Options
 
 let g:twiggy_num_columns            = get(g:,'twiggy_num_columns',            31                                                       )
+let g:twiggy_num_rows               = get(g:,'twiggy_num_rows',               31                                                       )
```

Comment:
I feel this is as good as any number in this range, but I don't have any strong feelings to adjusting the default here :)

Reply:
Ya, I'm thinking I want this to work like fugitive where it takes up half the screen, but I can't prove that is the actual behaviour looking through the code.  I'm fine with this for now.